ios/RNZendesk/RNZendesk.swift

```diff
@@ -204,4 +204,28 @@ class RNZendesk: RCTEventEmitter {
             }
         }
     }
+
+ @objc(getRequests:resolve:reject:)
+    func getRequests(status: String, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
+        DispatchQueue.main.async {
+            ZDKRequestProvider().getRequestsByStatus(status) { (result, error) in
+                if result != nil {
+                    let requestDicts = result!.requests.map { (request: ZDKRequest) -> [String: String] in
+                        var requestDict : [String:String] = [
+                            "id" : request.requestId,
+                            "status" : request.status,
+                            "subject" : request.subject!,
+                            "updatedAt": "\(Int(request.updateAt.timeIntervalSince1970) * 1000)",
```

Comment:
Can we just return this as an int rather than string?

Reply:
Initially, we weren't going to be using time since epoch and we using a string representation of the date.

While I agree it would be nicer to have this as an int, I'm not sure its worth the effort at this point to change the interface - especially when you take my non-existent knowledge of swift into account 😅